### ISSUE TITLE
Ubuntu 24.04 LTS "make" linking CXX executable error

### ISSUE THREAD

**Issue by umbertoragone** [lwvmobile/dsd-fme#262]
I am getting an error from CMake after compiling the project (it fails at 100%).

```
[100%] Linking CXX executable dsd-fme
/usr/bin/ld: /usr/local/lib/libsndfile.a(flac.c.o): undefined reference to symbol 'FLAC__stream_encoder_set_compression_level'
/usr/bin/ld: /lib/x86_64-linux-gnu/libFLAC.so.12: error adding symbols: DSO missing from command line

make[2]: *** [CMakeFiles/dsd-fme.dir/build.make:1385: dsd-fme] Error 1
make[1]: *** [CMakeFiles/Makefile2:84: CMakeFiles/dsd-fme.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

As mentioned in the title, I am trying to build dsd-fme on Ubuntu 24.04 LTS, installed very recently and with every package updated. I checked all the dependencies (a part from `libncurses5`, which cannot be found, but apt chose `libncurses-dev` instead), I tried both by cloning the git repo and using the download-and-install.sh script. Does anybody know how to resolve that error so I can link dsd-fme?

**Comment by lwvmobile** [comment]
Yeah, the ncurses libraries changed going forward to Ubuntu 24.04 LTS, I noticed that when I was working on M17-FME. Anyways, I just made a new auto installer script and left some notes in install for the packages. You can just download and run the new installer script with this. I just tested it out in a clean VM, and it worked just fine.

```
wget https://raw.githubusercontent.com/lwvmobile/dsd-fme/aw_dev/download-and-install-ubuntu2404lts.sh
sh download-and-install-ubuntu2404lts.sh
```

**Comment by umbertoragone** [comment]
Thanks for your quick reply, @lwvmobile. I tried the new dsd-fme installation script for Ubuntu 24.04 LTS in a VM and it worked flawlessly, but on my installation of Ubuntu it threw the same error as before, even after trying to remove and then re-install all the dependencies.

I ended up reinstalling Ubuntu since I didn't want to chase the problem too much, and now it works as expected. I managed to fully download and install everything. Thanks for your help and for the awesome dsd-fme project!
